helpers/controllers/authController.ts

```diff
@@ -84,7 +90,10 @@ export const logout = async (_parent: void, _: void, ctx: Context) => {
         error: 'Session Error'
       })
     }
-
+    ctx.res.setHeader(
+      'Set-Cookie',
+      `loggedIn=false; expires=${new Date(Date.now())}; Path=/`
+    )
```

Comment:
Why have a false value? Does it mean something else? Can't we simply clear the cookie on logout?